### PR TITLE
#169257391 setup swagger api docs

### PR DESCRIPTION
**Pull request by nshaphan**
#### What does this PR do?
Setup API documentation using swagger and JSdoc

#### Description of Task to be completed
- Installed swagger-JSdoc to retrieve JSdoc from routes files
- create API specifications file
- configure swagger UI routes
- modified routes files to use es6

#### How should this be manually tested?
- Clone the repo and do `npm install` to install dependencies
- Run `npm run start`
- visit `localhost:3000/api/docs` in browser

#### Any background context you want to provide?
- Port may change depending on your environment

#### What are the relevant pivotal tracker stories?
[#169257391](https://www.pivotaltracker.com/story/show/169257391)

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by b0nbon1 — DISMISSED** [review]
It looks okay.

**Comment by b0nbon1** [comment]
Please solve the conflicts and it will be okay

**Review by RaymondMwaura — DISMISSED** [review]
It works. Good job.

**Review by gildniy — DISMISSED** [review]
Looks nice

**Review by b0nbon1 — DISMISSED** [review]

**Review by timolinn — DISMISSED** [review]
LGTM

**Comment by timolinn** [comment]
@nshaphan fix conflicts

**Review by RaymondMwaura — APPROVED** [review]
This is good to merge.

**Review by b0nbon1 — APPROVED** [review]
Great work!

**Review by gildniy — APPROVED** [review]
it looks great

File: package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
   "scripts": {
     "test": "nyc mocha src/tests/*.js --require @babel/register",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "start": "node dist/index.js",
+    "start": "node -r dotenv/config dist/index.js",
```
Comment (timolinn) [inline package.json:16]: what does this `-r dotenv/config` do?

Comment (nshaphan) [reply package.json:16]: This load the variables from .env into process.env, so there is no need to import dotenv on every file it is needed

**Review by timolinn — APPROVED** [review]
LGTM

**Pull request merged**